receiver/sapnetweaverreceiver/documentation.md

```diff
@@ -2,44 +2,205 @@
 
 # sapnetweaverreceiver
 
-## Metrics
-
-These are the metrics available for this scraper.
-
-| Name | Description | Unit | Type | Attributes |
-| ---- | ----------- | ---- | ---- | ---------- |
-| **sapnetweaver.host.cpu_utilization** | CPU Utilization Percentage. | % | Gauge(Int) | <ul> </ul> |
-| **sapnetweaver.host.memory.virtual.overhead** | Virtualization System Memory Overhead. | bytes | Gauge(Int) | <ul> </ul> |
-| **sapnetweaver.host.memory.virtual.swap** | Virtualization System Swap Memory. | bytes | Gauge(Int) | <ul> </ul> |
-| **sapnetweaver.host.spool_list.used** | Host Spool List Used. |  | Sum(Int) | <ul> </ul> |
-| **sapnetweaver.icm_availability** | ICM Availability (color value from alert tree). |  | Sum(Int) | <ul> <li>control_state</li> </ul> |
-| **sapnetweaver.locks.enqueue.count** | Count of Enqueued Locks. | {locks} | Sum(Int) | <ul> </ul> |
```

Comment:
Just curious what is the reason for removing this metric?

Reply:
Went over with Sam and the swap space Percentage_Used, Free Memory and Configured Memory seemed more relevant for the key metric "Memory and Swap utilization"